Extension/lib/pages/log.js

```diff
@@ -403,8 +403,9 @@ PageController.prototype = {
 		var show = !this.searchRequest || StringUtils.containsIgnoreCase(filterData.requestUrl, this.searchRequest);
 		show &= this.searchTypes.length == 0 || this.searchTypes.indexOf(filterData.requestType) >= 0;
 		show &= !this.searchThirdParty || filterData.requestThirdParty;
-		show &= !this.searchWhitelisted || (filterData.requestRule && filterData.requestRule.whiteListRule);
-		show &= !this.searchBlocked || (filterData.requestRule && !filterData.requestRule.whiteListRule);
+		show &= ((!this.searchWhitelisted && !this.searchBlocked)
```

Comment:
@Mizzick i mean really, it should work the same way for third party check box.

Something like that:
```
var checkboxes = !(this.searchThirdParty || this.searchWhitelisted || this.searchBlocked);
checkboxes |= (this.searchThirdParty && filterData.requestThirdParty);
checkboxes |=  (this.searchWhitelisted && (filterData.requestRule && filterData.requestRule.whiteListRule));
checkboxes |= (this.searchBlocked && (filterData.requestRule && !filterData.requestRule.whiteListRule));
```